File: Codes/2824-count-pairs-whose-sum-is-less-than-target.cpp
/*
 * @Author: LetMeFly
 * @Date: 2023-11-24 10:00:19
 * @LastEditors: LetMeFly
 * @LastEditTime: 2023-11-24 10:01:51
 */
#ifdef _WIN32
#include "_[1,2]toVector.h"
#endif

class Solution {
public:
    int countPairs(vector<int>& nums, int target) {
        int ans = 0;
        for (int i = 0; i < nums.size(); i++) {
            for (int j = i + 1; j < nums.size(); j++) {
                ans += nums[i] + nums[j] < target;
            }
        }
        return ans;
    }
};

File: Codes/2824-count-pairs-whose-sum-is-less-than-target.py
'''
Author: LetMeFly
Date: 2023-11-24 11:04:43
LastEditors: LetMeFly
LastEditTime: 2023-11-24 11:11:37
'''
from typing import List

class Solution:
    def countPairs(self, nums: List[int], target: int) -> int:
        ans = 0
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                ans += nums[i] + nums[j] < target
        return ans